crates/gossip/src/gossip.rs

```diff
@@ -712,274 +712,6 @@ mod test {
         assert_eq!(op_1, received_ops[0]);
     }
 
-    #[tokio::test]
-    async fn respect_size_limit_for_new_ops() {
```

Comment:
These 3 tests are important but were relying on being able to run one gossip round and then do assertions. That's no longer reliable with the faster initiation. Replacing these with unit tests